### PR TITLE
Add Cordoning

### LINKED ISSUE #13: Missing cordon on outdated nodes
This script seems to be useful and in general do the right thing. But I think one step is missing for the smoothest operation: All the outdated nodes should be cordoned just before draining them. Otherwise pods might be migrated to other outdated nodes when a node is drained, which means then they would be migrated more than once during the rolling update.

### PR DESCRIPTION
**Pull request by cablespaghetti**
Add node cordoning and refactor things a little bit to give various options of what order things are run. (See README.md for explanation)

Fix #13 

This contains the changes in PR #15 as well, so it makes sense the address that one first. Sorry!

### CODE REVIEW

**Comment by crhuber** [comment]
@cablespaghetti great thanks. Okay, ill start with PR #15  then

**Comment by cablespaghetti** [comment]
This is slightly less huge now that other PR is merged. I've used this for a few clusters now without problems.

**Comment by crhuber** [comment]
@cablespaghetti  yup, i will test it early next week Thanks!

File: README.md [hunk 1]
```diff
@@ -96,11 +97,30 @@ eks-rolling-update.py -c my-eks-cluster
...
+| ASG_ORIG_MAX_CAPACITY_TAG | Temporary tag which will be saved to the ASG to store the state of the EKS cluster prior to update                 | eks-rolling-update:original_max_capacity |
 | CLUSTER_HEALTH_WAIT       | Number of seconds to wait after ASG has been scaled up before checking health of the cluster                       | 90                                   |
 | GLOBAL_MAX_RETRY          | Number of attempts of a health check                                                                               | 12                                   |
 | GLOBAL_HEALTH_WAIT        | Number of seconds to wait before retrying a health check                                                           | 20                                   |
+| BETWEEN_NODES_WAIT        | Number of seconds to wait after removing a node before continuing on                                               | 0                                    |
+| RUN_MODE                  | See Run Modes section below                                                                                        | 2                                    |
```
Comment (crhuber) [inline README.md]: The default is 2 actually

**Pull request merged**